browser/README.md

```diff
@@ -0,0 +1,124 @@
+# Azure Storage JavaScript Client Library for Browsers
+
+* Join the community discussion on Slack! [![Slack](https://azurestorageslack.azurewebsites.net/badge.svg)]( https://azurestorageslack.azurewebsites.net) 
+
+There are 5 generated JavaScript files for Azure Storage JavaScript Client Library:
+- `azure-storage.common.js` contains the common part for other 4 JavaScript files.
+- `azure-storage.table.js` contains the Azure Storage table service operation logic, which depends on azure-storage.common.js
+- `azure-storage.blob.js` contains the Azure Storage blob service operation logic, which depends on azure-storage.common.js
+- `azure-storage.queue.js` contains the Azure Storage queue service operation logic, which depends on azure-storage.common.js
+- `azure-storage.file.js` contains the Azure Storage file service operation logic, which depends on azure-storage.common.js
+
+We also provide samples to guide you quickly start with the Azure Storage JavaScript Client Library. Under your downloaded JavaScript Client Library zip file or [azure-storage-node/browser/samples](samples), you will find 4 HTML samples:
+- `sample-table.html` demostrates how to operate with Azure Storage table service in the browser
+- `sample-blob.html` demostrates how to operate with Azure Storage blob service in the browser
+- `sample-queue.html` demostrates how to operate with Azure Storage queue service in the browser
+- `sample-file.html` demostrates how to operate with Azure Storage file service in the browser
+
+After the JavaScript Client Library generation, you could open directly and try the samples with browsers such as Chrome/Edge/Firefox etc.
```

Comment:
After generating the JavaScript Client Library, you can try the samples in browsers such as Chrome/Edge/Firefox directly.